PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/sqlserver/task/CreateSqlServerTask.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+package com.microsoft.azure.toolkit.intellij.sqlserver.task;
+
+import com.microsoft.azure.management.Azure;
+import com.microsoft.azure.management.resources.ResourceGroup;
+import com.microsoft.azure.toolkit.intellij.common.Draft;
+import com.microsoft.azure.toolkit.lib.common.model.Region;
+import com.microsoft.azure.toolkit.lib.common.operation.AzureOperation;
+import com.microsoft.azure.toolkit.lib.sqlserver.SqlServerConfig;
+import com.microsoft.azure.toolkit.lib.sqlserver.model.SqlServerEntity;
+import com.microsoft.azure.toolkit.lib.sqlserver.service.AzureSqlServer;
+import com.microsoft.azure.toolkit.lib.sqlserver.service.ISqlServer;
+import com.microsoft.azuretools.ActionConstants;
+import com.microsoft.azuretools.authmanage.AuthMethodManager;
+import com.microsoft.azuretools.telemetry.TelemetryConstants;
+import com.microsoft.azuretools.telemetrywrapper.ErrorType;
+import com.microsoft.azuretools.telemetrywrapper.EventType;
+import com.microsoft.azuretools.telemetrywrapper.EventUtil;
+import com.microsoft.azuretools.telemetrywrapper.Operation;
+import com.microsoft.azuretools.telemetrywrapper.TelemetryManager;
+
+import java.util.Collections;
+
+public class CreateSqlServerTask {
+
+    private final SqlServerConfig config;
+
+    public CreateSqlServerTask(SqlServerConfig config) {
+        this.config = config;
+    }
+
+    @AzureOperation(
+        name = "sqlserver.create",
+        params = {
+            "config.getServerName()",
+            "config.getSubscription().displayName()"
+        },
+        type = AzureOperation.Type.SERVICE
+    )
+    public ISqlServer execute() {
+        final Operation operation = TelemetryManager.createOperation(ActionConstants.MySQL.CREATE);
+        try {
+            operation.start();
+            final String subscriptionId = config.getSubscription().subscriptionId();
+            EventUtil.logEvent(EventType.info, operation, Collections.singletonMap(TelemetryConstants.SUBSCRIPTIONID, subscriptionId));
+            // create resource group if necessary.
+            if (config.getResourceGroup() instanceof Draft) {
```

Comment:
this piece of code shall soon be duplicate: 
config.getResourceGroup() instanceof Draft 
...

Reply:
remember to merge them